modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java

```diff
@@ -1061,6 +1063,60 @@ private void _assertCPOptionCategory() throws Exception {
 			cpOptionCategory.getTitle(LocaleUtil.getSiteDefault()));
 	}
 
+	private void _assertDataDefinition() throws Exception {
+		DataDefinitionResource.Builder dataDefinitionResourceBuilder =
+			_dataDefinitionResourceFactory.create();
+
+		DataDefinitionResource dataDefinitionResource =
+			dataDefinitionResourceBuilder.user(
+				_serviceContext.fetchUser()
+			).build();
+
+		DataDefinition dataDefinition =
+			dataDefinitionResource.
+				getSiteDataDefinitionByContentTypeByDataDefinitionKey(
+					_serviceContext.getScopeGroupId(), "journal",
+					"test-data-definition-1");
+
+		Assert.assertNotNull(dataDefinition);
```

Comment:
since we already know that if data definition doesn't exist it will throw an exception we can remove this statement
```suggestion

```